### PR TITLE
Theme updated and README added

### PR DESCRIPTION
**Pull request by Urbork**
# Theme 
- colors unified 
- font weight updated 
- breakpoints changed to small/medium/large 

# README 
- temporary image, will be changed later 
- demo link 
- features 
- technologies
- contributors 

## and some simple changes in movies section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+
+## 🎉Contributors
+
+- [Mariusz Matusiewicz](https://github.com/mariuszmmm)
+- [Adrian Woźniak](https://github.com/Adrode)
+- [Jonasz Jachym](https://github.com/Urbork)
+
+---
+
 # Getting Started with Create React App
```
Comment (Adrode) [inline README.md:44]: Nice and creative description. Good job!

File: README.md [hunk 2]
```diff
@@ -44,27 +82,3 @@ You don't have to ever use `eject`. The curated feature set is suitable for smal
...
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (Adrode) [inline README.md:70]: Good, that you've removed this

File: src/components/MovieTile/MovieRating/index.js [hunk 3]
```diff
@@ -9,11 +9,11 @@ export const MovieRating = ({ big, rating, votes }) => (
   <>
     {rating ? (
       <>
-        <MovieTileRating>
+        <MovieTileRating big={big}>
```
Comment (Adrode) [inline src/components/MovieTile/MovieRating/index.js:12]: I think you don't even need to use this syntax - big={big}. Just "big" as a parameter should be enough (not sure, but i've tried it on my branch and it works)

File: src/components/MovieTile/MovieRating/index.js [hunk 4]
```diff
@@ -9,11 +9,11 @@ export const MovieRating = ({ big, rating, votes }) => (
...
           <StyledStarIcon />
-          {rating.toFixed(1)}{" "}
+          {rating.toFixed(1).replace(".", ",")}{" "}
           {big ? <MovieTileRatingTen>/ 10</MovieTileRatingTen> : ""}
-          <MovieTileRatingVotes>{votes} votes</MovieTileRatingVotes>
+          <MovieTileRatingVotes big={big}>{votes} votes</MovieTileRatingVotes>
```
Comment (Adrode) [inline src/components/MovieTile/MovieRating/index.js:16]: Same as comment above

File: src/components/MovieTile/styled.js [hunk 5]
```diff
@@ -29,13 +31,14 @@ export const MovieTileLink = styled.a`
 
 export const MovieTileImage = styled.img`
   margin: 0;
-  width: 292px;
+  /* width: 292px; */
```
Comment (Adrode) [inline src/components/MovieTile/styled.js:34]: You can remove this, I think

**Review by Adrode — APPROVED** [review]
Code looks good, appreciate your work!

**Review by mariuszmmm — APPROVED** [review]
Great job.

File: src/components/MovieTile/MovieRating/index.js (hunk 3 above)
Comment (Urbork) [reply src/components/MovieTile/MovieRating/index.js:12]: You are probably right. Started with just "big" and don't even remember why I changed it. Thank you

**Pull request merged**